### PR TITLE
Add a new Label to ListTile for more than 3 lines of text.

### LINKED ISSUE #83: New line wrap for some list items
Probably since the introduction of the left vertical category name, we now have some list items which show an extra new line effect.  As a result the favs counter is shown in a new line as shown below.

image

### PR DESCRIPTION
**Pull request by abhinayagarwal**
This fixes #83 - line wrap problem for list items in session catalogue

### CODE REVIEW

File: DevoxxClientMobile/src/main/java/com/devoxx/views/cell/ScheduleCell.java [hunk 1]
```diff
@@ -135,8 +135,14 @@ private void updateListTile() {
         listTile.setTextLine(2, DevoxxBundle.getString("OTN.SCHEDULE.IN_AT",
                 session.getRoomName(),
                 DevoxxSettings.TIME_FORMATTER.format(session.getStartDate()),
-                DevoxxSettings.TIME_FORMATTER.format(session.getEndDate())) +
-                (showDate ? "\n" + DevoxxSettings.DATE_FORMATTER.format(session.getStartDate()) : ""));
+                DevoxxSettings.TIME_FORMATTER.format(session.getEndDate())));
+        
+        if (showDate) {
+            final Label label = new Label(DevoxxSettings.DATE_FORMATTER.format(session.getStartDate()));
```
Comment (jperedadnr) [inline DevoxxClientMobile/src/main/java/com/devoxx/views/cell/ScheduleCell.java]: Avoid creating the label all the time, use a global field

File: DevoxxClientMobile/src/main/resources/com/devoxx/common.css [hunk 2]
```diff
@@ -249,17 +249,22 @@
     -fx-font-size: 1.05em;
     -fx-wrap-text: false;
 }
+
+.schedule-cell > .list-tile > .text-box > .extra-text,
```
Comment (jperedadnr) [inline DevoxxClientMobile/src/main/resources/com/devoxx/common.css]: I'd probably use a different styling for the extra text. With so many lines now, it is hard to read.

Comment (abhinayagarwal) [reply DevoxxClientMobile/src/main/resources/com/devoxx/common.css]: This is how the cell looks with the extra label at the bottom:

image

Comment (abhinayagarwal) [reply DevoxxClientMobile/src/main/resources/com/devoxx/common.css]: With different styling do you want to use a different color for the text or alter the text size?

Comment (jperedadnr) [reply DevoxxClientMobile/src/main/resources/com/devoxx/common.css]: Personally, I'd use lighter color and smaller font size.

Comment (abhinayagarwal) [reply DevoxxClientMobile/src/main/resources/com/devoxx/common.css]: How about this?

image

Comment (jperedadnr) [reply DevoxxClientMobile/src/main/resources/com/devoxx/common.css]: Looks fine to me.

**Pull request merged**